packages/reports/src/app.tsx

```diff
@@ -1,11 +1,18 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import Routes from 'Containers/routes.jsx';
 import { MobxContentProvider } from 'Stores/connect';
 import 'Sass/app.scss';
 import initStore from './init-store'; // eslint-disable-line import/extensions
+import type { TRootStore } from 'Stores/index';
```

Comment:
@farzin-deriv the `TRootStore` type is already utilizing `TStores` and extending it with necessary modules. Don't you think we better declare it in one place and import, rather than import and extend TStores in multiple files?

Reply:
@akmal-deriv The type of the `root_store` we get from the `core` inside other packages is `TStores`, Then inside each package, we extend it to add additional stores.

Here we get `TStores` from core in `App` props and then pass it to `initStore` method at line `16` to extend it to be `TRootStore`.